docs/en/observability/uptime-set-up-app.asciidoc

```diff
@@ -0,0 +1,165 @@
+[[uptime-set-up-choose-agent]]
+= Use the {uptime-app}
+
+beta[]
+
+You can now create monitors directly in the {uptime-app}.
+To try this beta functionality, first you need to enable Monitor Management:
+
+. Go to **{observability}** -> **Uptime**.
+. Click **Monitor Management**.
+. Review the terms and conditions and click **Enable** to start using Monitor Management.
+
+From here, you can either <<global-managed-testing-infrastructure,use Elastic's global managed testing infrastructure>>
+or <<private-locations,create a private location>> to run monitors from your own premises.
+
+NOTE: To use the Elastic Synthetics integration, see the https://www.elastic.co/guide/en/observability/8.3/uptime-set-up.html#uptime-set-up-choose-agent[8.3 documentation]. The Elastic Synthetics integration is similar to the {uptime-app} approach, but contains fewer configuration options.
+
+[discrete]
+[[global-managed-testing-infrastructure]]
+== Use global managed testing infrastructure
+
+With Elastic's global managed testing infrastructure, you can create and run monitors in multiple
+locations using the {uptime-app} UI without having to manage your own infrastructure.
+Elastic takes care of software updates and capacity planning for you.
+
+To add a monitor using Elastic's global managed testing infrastructure:
+
+. Go to the **Monitor Management** page.
+. Click **Add monitor**.
+. In _Monitor locations_, select one or more locations.
+. Choose a _Monitor type_ and configure the monitor as needed.
+. Click **Save monitor**.
+
+If you are setting up a **Browser** monitor, see <<synthetics-create-test>>
+for more details on writing synthetic tests.
+
+[discrete]
+[[private-locations]]
+== Run a monitor on private locations
+
+Private locations allow you to run monitors from your own premises.
+	
+TIP: You can run the same monitor both from Elastic's global managed infrastructure _and_ your private locations to compare results.
+
+Before you create a private location, you need to set up {fleet}, and connect an agent to it.
+Then, you can create a private location in the {uptime-app} UI.
+Now, whenever you access the monitor creation page, this private location will appear in the list
+of locations to which you can deploy monitors.
+
+[discrete]
+[[private-locations-fleet-agent]]
+=== Set up {fleet-server} and {agent}
+
+To deploy monitors to private locations, you must complete two prerequisites:
+
+* *Set up {fleet-server}*: If you are using {ecloud}, {fleet-server} will already be provided and you can skip this step.
+To learn more, see <<set-up-fleet,{fleet}>>.
+* **Create an agent policy**:  For more information on agent policies and creating them,
```

Comment:
Somewhere in here we should note that we currently only support a single agent per policy and do not support load balancing if multiple agents are added to the same policy. Doing so may lead to weird behavior in fact.

Reply:
Good point @andrewvc, @colleenmcginnis we should add something about this, for example (feel free to change):

> A Private Location should be set up against an Agent Policy that runs on a single Elastic Agent. Do not run the same Agent Policy on multiple Agents being used for Private Locations, as you may end up with duplicate or missing tests. Private Locations do not currently load balance tests across multiple Elastic Agents.

@andrewvc as we're including something about this, should we also include details of how the Agent handles concurrency, for example something like (again, feel free to change):

> If there are multiple monitors that need to run from a Private Location at the same time, up to two will run concurrently on that Elastic Agent. The other monitors will wait until a previos test has finished running and there is capacity, up to a maximum of two at the same time.